doc/newsfragments/1937_changed.dict_match_for_ignored_fields.rst

```diff
@@ -0,0 +1 @@
+:py:meth:`dict.match <testplan.testing.multitest.result.DictNamespace.match>` should also compare the fields even their keys are marked as "ignored", thus for this assertion the nested values in a dictionary can be displayed in a compact view on Web UI (comparable fields are placed in the same row).
```

Comment:
dict.match should compare "ignored" fields -- it doesn't make sense when reading from user perspective right?

btw, avoid "should" in news fragment -- what does it mean? it makes the announcement sounds like a issue description.